package.json

```diff
@@ -18,30 +18,39 @@
 	"dependencies": {
 		"@hookform/resolvers": "^3.6.0",
 		"@reduxjs/toolkit": "^2.2.5",
+		"@testing-library/user-event": "^14.5.2",
 		"axios": "^1.7.2",
+		"axios-mock-adapter": "^1.22.0",
```

Comment:
check if this library is being or used, if not remove it  "axios-mock-adapter": "^1.22.0",